packages/terra-framework-docs/src/terra-dev-site/doc/infinite-list/guides.2/InfiniteListSections.1.doc.mdx

```diff
@@ -1,10 +1,31 @@
 import { Badge } from 'terra-infinite-list/package.json?dev-site-package';
+import { Notice } from '@cerner/terra-docs';
 
 import InfiniteListSection from '../guides/InfiniteListSection';
 
 <Badge />
 
-# Terra InfiniteList - Sections
+[![Deprecated](https://badgen.net/badge/status/Deprecated/grey)](https://github.com/cerner/terra-framework/blob/main/packages/terra-infinite-list#readme)
+
+# Terra Infinite List — Sections
+
+<Notice variant='deprecation' >
+
+The terra-infinite-list component has been deprecated and is no longer recommended for use.
+
+From a usability perspective, infinite scrolling is a pattern designed and best used when users are casually looking for or discovering things, like shopping and social media. The infinite scrolling pattern should not be used for [goal-oriented tasks](https://www.nngroup.com/articles/infinite-scrolling/). Cerner products and our users most often fall into goal-oriented workflows. Users should have proper navigation methods (pagination) to navigate to predetermined points within the content easily.
+
+Infinite scrolling also poses significant accessibility challenges for multiple user groups:
+
+* Keyboard-only users have to navigate the entire infinite scroll to get to the content below or to the side of the endless loading content. This creates a very time-consuming and unnecessary burden for the keyboard-only user.
+* Screen reader users would have the same problem even though they have more ways to navigate content on the page. When content is dynamically loaded and not within the DOM, screen reader users won't have access to any of that content which poses more significant challenges than just navigating.
+* Focus can be lost when new content loads. Unless focus is managed, sighted keyboard-only users will have difficulties finding where they are on the page.
+* For data with hierarchical relationships, infinite scrolling can make it difficult for screen readers to effectively communicate the required context if related content is split between loaded and unloaded content.
+
+The [terra-paginator](/components/terra-paginator/paginator/paginator) should be considered as an alternative approach to paging large amounts of data.
```

Comment:
The `/components/terra-paginator/paginator/paginator` relative path may work in terra-ui but not in the terra-framework dev site since terraa-paginator is in terra-core. Suggest to use absolute path.

Reply:
Updated https://github.com/cerner/terra-framework/pull/1553/commits/45f3911d6b1e7e15be638c55134b864546f02c47